src/date-range-picker/utils.tsx

```diff
@@ -1,15 +1,20 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import { joinDateTime, splitDateTime } from '../internal/utils/date-time';
+import { isIsoDateOnly } from '../internal/utils/date-time';
 import { normalizeTimeString } from '../internal/utils/date-time/join-date-time';
 import { DateRangePickerProps } from './interfaces';
-import { setTimeOffset } from './time-offset';
+import { setTimeOffset, shiftTimeOffset } from './time-offset';
 
 export function formatValue(
   value: null | DateRangePickerProps.Value,
   { timeOffset, dateOnly }: { timeOffset: { startDate?: number; endDate?: number }; dateOnly: boolean }
 ): null | DateRangePickerProps.Value {
-  if (!value || value.type === 'relative') {
+  if (
+    !value ||
+    value.type === 'relative' ||
+    (value.endDate === '' && value.endDate === '' && value.type === 'absolute')
```

Comment:
We're checking `endDate` here twice, I assume one of these should be `startDate`